EIPS/eip-2025.md

```diff
@@ -11,56 +11,58 @@ requires: 1890
 ---
 
 ## Simple Summary
-Add  `0.044 ETH` per block for 18 months as a developer block reward reserved for funding Ethereum1.X development.
+Add `0.044 ETH` per block for 18 months as a developer block reward reserved for funding Ethereum 1.X development.
```

Comment:
> Add 0.044 ETH per block for 18 months as a developer block reward reserved for funding Ethereum1.X development

How do we define "Ethereum1.X development"? Should we say more about this or try to narrow the scope? Or just explicitly leave it up to the discretion of the governors/keyholders?